main.go

```diff
@@ -69,7 +79,11 @@ func chanHandler(feed *rss.Feed, newchannels []*rss.Channel) {
 func makeHandler(master *feeds.Feed, sourceName string) rss.ItemHandlerFunc {
 	return func(feed *rss.Feed, ch *rss.Channel, items []*rss.Item) {
 		for i := 0; i < len(items); i++ {
-			published, _ := items[i].ParsedPubDate()
+			published, err := items[i].ParsedPubDate()
+			if err != nil {
+				log.Fatal(err)
```

Comment:
It may be worth considering skipping over this item if the date cannot be parsed? 

If we add `continue` when `err != nil` then we avoid the risk of interacting with an invalid date on line 89 (`if published.After(weekAgo)`)